ixmp/core.py

```diff
@@ -286,6 +286,31 @@ def add_region_synonym(self, region, mapped_to):
 
         self._backend.set_node(region, synonym=mapped_to)
 
+    def timeslices(self):
+        """Return all subannual time slices defined for the IAMC-style
+           timeseries format.
+
+        Returns
+        -------
+        :class:`pandas.DataFrame`
```

Comment:
Sphinx auto-adds the ReST syntax here, so you can just write `pandas.DataFrame`.

Reply:
That could be done to all other places simultaneously as I just followed to some other fragment of code.